font-icons/overview.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Overview
+page_title: .NET MAUI Font Icons - Overview
+description: Telerik font icons are collections of small vector graphics used across the components in the Telerik UI for .NET MAUI suite.
+slug: telerik-font-icons
+position: 0
+---
+
+# Telerik UI for .NET MAUI Font Icons
+
+Telerik font icons represents collection of small vector graphics used across the components from Telerik UI for .NET MAUI suite. Examples of using font icons include the expand and collapse indicators of Accordion, Expander and TreeView controls, filter and sort indicators of DataGrid control, and other.
+
+This article will give an overview on how you can utilize the Telerik font icons in your .NET MAUI app. 
+
+## Include the Required Telerik Font File
+
+1. The Telerik Font Icons are located in the `telerikcontrolsicons.ttf` and `telerikfontexamples.ttf` files. There are two options you could use to get the `.ttf` file.
```

Comment:
Note that you're spelling Telerik Font Icons inconsistently. In the first two occurrences you chose to capitalize only Telerik, while here it's Telerik Font Icons, implying a product name or trademark. According to [this list](https://www.progress.com/legal/trademarks), Telerik Font Icons is not an actual trademark or approved product name, so I would suggest going with `Telerik font icons`, unless in a section heading or page title.